config.js

```diff
@@ -90,15 +92,99 @@ smtpServer.messageBody = function () {
 	// getMessageCache() intoarce calea catre fisierul temporar in care e corpul mesajului
 	// client.sendMessage() intoarce SmtpStatus
 };
+*/
+
+smtpServer.smtpInit = function() {
+	var ret = new Array();
+	ret['code'] = 250;
+	ret['message'] = "init from JS";
+	ret['disconnect'] = false;
+	return ret;
+}
+
+smtpServer.smtpAuth = function() {
+	var ret = new Array();
```

Comment:
You are creating an array type object and then adding some properties to it. It's better to create a plain object and leave the array out of the equation. You can use JSON to do this. Example:
```javascript
return {
    "code": 250,
    "message": "auth from JS",
    "disconnect": false
};
```